examples/ftp-service-read-write/ftp_service_read_write.md

```diff
@@ -1,13 +1,13 @@
 # FTP service - Read/Write file
 
-The FTP service is used to receive file/directory changes that occur in a remote location using the FTP protocol. This sample includes receiving file/directory related change events from a listener and using the `append` api of the `ftp:Caller` to interact with the FTP server.
+The `ftp:Service` connects to a given FTP server via the `ftp:Listener` and then sends and receives files as byte streams via the `ftp:Caller`. The `ftp:Caller` is available as a parameter in the `onFileChange` method which is created with the configurations provided to the `ftp:Listener`. Once the `onFileChange` is invoked, the `append` method of FTP caller can be used to append a file to the files existing in the FTP server. Use this to listen to file changes occurring in a remote file system and do CRUD operations with the remote server.
 
```

Comment:
The `ftp:Service` connects to a given FTP server via the `ftp:Listener`. Once connected, service starts receiving events every time a file deleted or added to the server. To take actions for these events `ftp:Caller` is used. The `ftp:Caller` can be specified as a parameter of `onFileChange` remote method. The `ftp:Caller` allows interacting with the server via `append`, `foo`, `bar`, etc remote methods. Use this to listen to file changes occurring in a remote file system and take action for those changes.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/3749/commits/6f93006b56680b05378562e2b52a31c73ff37fd6